ls1intum/Artemis
# `Text exercises`: Fix feedback visualization for grading instructions with 0 points

### Checklist
#### General
#### Client

### Motivation and Context
This error was discovered by @willergeorg while testing #4756. When using grading instructions with 0 points, the feedback for the student is not displayed correctly (see Screenshot). 

### Description
The logic for determining whether or not a feedback block is empty did not take grading instructions into account. When using grading instructions, the feedback can have text even without adding another detail text. The newly added hasContent()-method allows the check for this case. 

### Steps for Testing
Prerequisites:
- 1 Exam with a text exerise
- 1 course text exercise
- Both exercises need grading instructions with 0 points

1. Participate in the exercises
2. Assess the submission using the grading instructions.
3. View the result as a student. Make sure that every feedback has a correct background color.

### Review Progress
#### Code Review
#### Manual Tests


### Screenshots
old:
![grafik](https://user-images.githubusercontent.com/26540346/156900350-50899b77-a588-4b28-a76f-016a36a1b0ce.png)

new:
![grafik](https://user-images.githubusercontent.com/26540346/156900356-d8ca2292-d51e-4007-897f-eb27d07c4747.png)
